packages/server/lib/browsers/index.ts

```diff
@@ -155,13 +156,36 @@ export = {
     // TODO: normalizing opening and closing / exiting
     // so that there is a default for each browser but
     // enable the browser to configure the interface
-    instance.once('exit', () => {
+    instance.once('exit', async (code, signal) => {
       ctx.browser.setBrowserStatus('closed')
       // TODO: make this a required property
       if (!options.onBrowserClose) throw new Error('onBrowserClose did not exist in interactive mode')
 
+      const browserDisplayName = instance?.browser?.displayName || 'unknown'
```

Comment:
Would Electron be the fallback here or when would we not know this value? User defied browser instance?

Reply:
I'm really not sure in what cases we don't have the display name, just following typescripts lead to be honest. If we don't have a browser or a display name available i honestly don't know what could be happening.